tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -756,6 +756,7 @@ private function getDatasetsForOptions(): array
             'ean13' => 'ean13_2',
             'mpn' => 'mpn_2',
             'reference' => 'reference_2',
+            'date_new' => date('Y-m-d'),
```

Comment:
Maybe you should use a hard-coded value instead of a dynamic one here, just in case there is an offset of a few milliseconds that make the two dates different, even if the chances are slim it could cause some instability